src/go.mod

```diff
@@ -3,17 +3,22 @@ module github.com/containers/toolbox
 go 1.13
 
 require (
-	github.com/HarryMichal/go-version v1.0.0
+	github.com/HarryMichal/go-version v1.0.1
```

Comment:
Thanks for splitting the dependency bumps into two steps. It might came in handy in case we hit any regressions.